Makefile

```
@@ -385,3 +395,12 @@ check-git-diff: update
 .PHONY: docs
 docs: crd-ref-docs
 	$(REFERENCE_DOC_GENERATOR) --source-path=api/v1alpha1/ --renderer=markdown --config=hack/docs/config.yaml --output-path=docs/api_reference.md
+
+## perform vulnerabilities scan using govulncheck.
+.PHONY: govulnscan
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)